tests/network-tests/src/flows/proposals/exactExecutionBlock.ts

```diff
@@ -22,12 +23,12 @@ export default async function exactExecutionBlock({ api, query, lock }: FlowProp
   const createProposalFixture = new CreateProposalsFixture(api, query, [
     {
       type: 'Signal',
-      details: `Proposal to be executed at block ${exactExecutionBlock}`,
+      details: (`Proposal to be executed at block ${exactExecutionBlock}` as unknown) as Bytes,
       asMember: memberId,
       title: `Executes at #${exactExecutionBlock}`,
       description: `Proposal to be executed at block ${exactExecutionBlock}`,
       exactExecutionBlock,
-    },
+    } as ProposalCreationParams<'Signal'>,
```

Comment:
Avoid type assertions

Reply:
Addressed in [b552fba](https://github.com/Joystream/joystream/pull/3997/commits/b552fba6a40a75129fb358c4b771150313077472)